src/Caliburn.Micro.Platform/Caliburn.Micro.Platform.csproj

```diff
@@ -74,7 +80,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Remove="Platforms\net46\AssemblyInfo.cs" />
+    <None Remove="Platforms\net46-netcore\AssemblyInfo.cs" />
```

Comment:
Not related to this PR, but I suspect the whole `ItemGroup` can be removed. VS tends to screw up projects when removing files that are wildcard included.

Reply:
Removed the ItemGroup in df87adaac6208f07ea8fd727e5c2a2eed75512de